### PR TITLE
feature/regex for text processing

### PR DESCRIPTION
**Pull request by sayv-xo**
implemented feature to handle regex operations

### CODE REVIEW

**Review by hale-Dennis — APPROVED** [review]
Great

File: src/main/java/org/group2/textprocessingtool/controllers/TextProcessingMainController.java [hunk 1]
```diff
@@ -2,15 +2,19 @@
...
 import javafx.event.ActionEvent;
 import javafx.fxml.FXML;
-import javafx.scene.control.Alert;
-import javafx.scene.control.TextArea;
-import javafx.scene.control.TextField;
-import javafx.scene.control.TextInputDialog;
+import javafx.geometry.Insets;
+import javafx.scene.control.*;
+import javafx.scene.layout.GridPane;
 import javafx.stage.FileChooser;
```
Comment (arkohaddoebenezer) [inline src/main/java/org/group2/textprocessingtool/controllers/TextProcessingMainController.java:8]: Please are you sure this ought to be removed?? are they not being used on other branches?

**Review by arkohaddoebenezer — CHANGES_REQUESTED** [review]

File: src/main/java/org/group2/textprocessingtool/controllers/TextProcessingMainController.java (hunk 1 above)
Comment (arkohaddoebenezer) [reply src/main/java/org/group2/textprocessingtool/controllers/TextProcessingMainController.java:8]: > Please are you sure this ought to be removed?? are they not being used on other branches?

Screenshot (337)

**Review by arkohaddoebenezer — APPROVED** [review]
we import them incase it breaks the code

**Pull request merged**